server/test/officeTest.js

```diff
@@ -0,0 +1,42 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+
+import server from '../app';
+
+// eslint-disable-next-line no-unused-vars
+const should = chai.should();
+
+chai.use(chaiHttp);
+
+
+describe('/POST office', () => {
+  const correctDetails = {
+    type: 'logo.jpg',
+    name: 'office name',
+  };
+  const wrongDetails = {
+    name: 'party name',
+  };
+  
+  it('it should be able to POST a party with all fields being entered', (done) => {
+    chai.request(server)
+      .post('/api/v1/offices')
+      .send(correctDetails)
+      .end((request, response) => {
+        response.body.statusMessage.should.equal('Office created successfully');
+        response.should.have.status(201);
+        done();
+      });
+  });
+  
+  it('should not post an Office with a missing field', (done) => {
```

Comment:
@chiboycalix great job. Best practice for writing tests requires that you test for edge cases first before testing when correct data is supply. It would make sense if the test for incomplete detail come before the test for complete details. How about that?

Reply:
Okay, I never thought of that, I'll do that going forward. It's actually intuitive to do it this way. Thank you for point this out to me.